### PR TITLE
Implement places page

### PR DESCRIPTION
**Pull request by HelenaEksler**
#125 
Follows up #116 #121 #122

### CODE REVIEW

**Comment by HelenaEksler** [comment]
selection_190
selection_191

**Comment by HelenaEksler** [comment]
@bitamar PR is ready for the code review.

File: haskala/modules/custom/haskala_terms/plugins/content_types/terms_list/terms_list.inc [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+  // Get hebrew alphabet;
+  $letters_he = haskala_book_hebrew_alphabet_without_sofit();
+
+  $count = 0;
+  $blocks = array();
+  $limit = 50;
+
+  // Get 50 terms of certain vocabulary, starting from current letter. If there
+  // are less then 50 items - take terms starting from next letter by alphabet.
+  do {
```
Comment (bitamar) [inline haskala/modules/custom/haskala_terms/plugins/content_types/terms_list/terms_list.inc]: This duplicates code from the books plugin. Please create a function that does this loop.

**Comment by bitamar** [comment]
@HelenaEksler - Please add a simple behat test that visits the places page

**Comment by HelenaEksler** [comment]
Closed.

**Pull request closed without merging**